### ISSUE TITLE
LSFJobManager doesn't parse date correctly

### ISSUE THREAD

**Issue by askask** [TheRoddyWMS/BatchEuphoria#128]
Exception
```
2018-08-21 11:13:57,775 [pool-7-thread-1] WARN  cluster.ClusterJobManager  - Text 'Aug 21 11: 2018' could not be parsed at index 10
2018-08-21 11:13:57,775 [pool-7-thread-1] WARN  cluster.ClusterJobManager  - java.time.format.DateTimeFormatter.parseResolved0(DateTimeFormatter.java:1949)
java.time.format.DateTimeFormatter.parse(DateTimeFormatter.java:1851)
java.time.ZonedDateTime.parse(ZonedDateTime.java:597)
de.dkfz.roddy.execution.jobs.cluster.lsf.LSFJobManager.parseTime(LSFJobManager.groovy:133)
```

Output of `bjobs`:
```
{
  "COMMAND":"bjobs",
  "JOBS":1,
  "RECORDS":[
    {
      "JOBID":"268323",
      "JOB_NAME":"r180821_111215186_pid_4_coveragePlotSingle",
      "STAT":"DONE",
      "USER":"otptest",
      "QUEUE":"devel",
      "JOB_DESCRIPTION":"",
      "PROJ_NAME":"default",
      "JOB_GROUP":"",
      "JOB_PRIORITY":"",
      "PIDS":"15005,15026,15028,15164,15942,15943",
      "EXIT_CODE":"",
      "FROM_HOST":"redacted",
      "EXEC_HOST":"redacted",
      "SUBMIT_TIME":"Aug 21 11:12",
      "START_TIME":"Aug 21 11:13",
      "FINISH_TIME":"Aug 21 11:13",
      "CPU_USED":"00:00:21",
      "RUN_TIME":"00:00:28",
      "USER_GROUP":"",
      "SWAP":"",
      "MAX_MEM":"380 Mbytes",
      "RUNTIMELIMIT":"",
      "SUB_CWD":"$HOME",
      "PEND_REASON":"",
      "EXEC_CWD":"redacted",
      "OUTPUT_FILE":"redacted",
      "INPUT_FILE":"",
      "EFFECTIVE_RESREQ":"select[type == local] order[r15s:pg] rusage[mem=51.00] ",
      "EXEC_HOME":"redacted",
      "SLOTS":"4",
      "ERROR_FILE":"",
      "COMMAND":"redacted",
      "DEPENDENCY":"done(268322)"
    }
  ]
}
```
with bjobs version `IBM Spectrum LSF 10.1.0.0 build 459939, Jul 6 2017`

**Comment by vinjana** [comment]
Notice that the string that is attempted to be parsed is not a valid time specification. Seems like this is an early stage parse error, such that produced the wrong time string that then later fails to get parsed. I would check whether the JSON is correctly parsed into map/list structure.

**Comment by askask** [comment]
Should be finally fixed with 65ba6e0d4ea0af76d08248e74c8723c656bd5a6e.
